### ISSUE TITLE
模型是否能像MonoScene一样直接从图像端到端预测？

### ISSUE THREAD

**Issue by 123456bfb** [hustvl/Symphonies#15]
你好，我发现训练好的模型在推理的时候仍然需要输入预处理的深度预测.npy数据，是否能像MonoScene一样直接从图像端到端预测？

**Comment by npurson** [comment]
I'm a bit unsure about the exact query you're looking for.

If you want the model to predict depth directly from images without needing preprocessed depth prediction data, you can simply add an additional depth prediction network within the model for online inference. The reason why we currently use the preprocessed data is for training efficiency and settings alignment.

If you're referring to the inability to backpropagate gradients "end-to-end" with the proposed VPL, we've recently experimented with using LSS instead of VPL, which also works. This may be released with our upcoming work.

I hope this addresses your query. :)

**Comment by 123456bfb** [comment]
感谢您的回复，我想问你的就是VPL模块不能从图像端到端进行。
